lib/Ix/DBIC/ResultSet.pm

```diff
@@ -893,4 +893,252 @@ sub ix_set ($self, $ctx, $arg = {}) {
   return @$ret;
 }
 
+sub ix_get_list ($self, $ctx, $arg = {}) {
+  my $rclass = $self->_ix_rclass;
+  $ctx = $ctx->with_account($rclass->ix_account_type, $arg->{accountId});
+
+  my $key = $rclass->ix_type_key;
+  my $key1 = $rclass->ix_type_key_singular;
+  my $fetch_arg = "fetch\u$key";
+
+  my $schema = $ctx->schema;
+
+  my $limit = $arg->{limit} // 500;
+  $limit = 500 if $limit > 500 && $arg->{$fetch_arg};
+
+  my $search = $self->_ml_search_args($ctx, $arg);
+  $search->{filter}{'me.isActive'} = 1;
+
+  my $search_page = $search->{rs}->search(
+    $search->{filter},
+    {
+      $search->{sort}->%*,
+      rows      => $limit,
+      offset    => $arg->{position} // 0,
+      result_class => 'DBIx::Class::ResultClass::HashRefInflator',
+    },
+  );
+
+  my @rows = $search_page->all;
+  my @ids  = map {; $_->{id} } @rows;
```

Comment:
Right, I thought that was a weird thing to do. Will be much faster to get rid of it (unless fetch* is true for the object of the same type)